sourcegraph/about
# Swap GE logo for Redfin

Closes #5893 - Swaps out GE logo for Redfin on CustomerLogos component

### Changelog
""

### Test
1. Ensure prettier has standardized the proposed changes.
2. Ensure no breaking changes on `CustomerLogos`
